lib/mocha.js

```diff
@@ -240,6 +242,24 @@ Mocha.prototype.invert = function(){
 };
 
 /**
+ * Apply randomization to the tests
+ *
+ * @param {String} mode
+ * @param          seed
+ * @return {Mocha}
+ * @api public
+ */
+
+Mocha.prototype.randomize = function(mode, seed){
+  if (mode !== 'tests' /* && mode !== 'suites' && mode !== 'both' */) {
+    throw new Error('unrecognized randomization mode "' + mode + '"');
+  }
+  this.options.randomizeMode = mode;
+  this.options.randomizeSeed = seed != null ? seed : Math.random();
```

Comment:
If this module does not automatically a seed with sufficient entropy then I'd prefer not to use it.  If it doesn't, please use [random-js](https://www.npmjs.com/package/random-js).  

```js
var engine = Random.engines.mt19937().autoSeed();
var randomTest = Random.pick(engine, tests);
```